test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -23,6 +24,50 @@ public class AggregatorTest
     {
         private readonly AggregatorStore aggregatorStore = new("test", AggregationType.HistogramWithBuckets, AggregationTemporality.Cumulative, 1024, Metric.DefaultHistogramBounds);
 
+        [Fact]
+        public void CounterWithTags()
```

Comment:
lets use existing tests model, and add new use case with null tags. This is not specific to Counters, but applicable to every instrument.

Reply:
Do you mean adding `[InlineData()]` cases to `MetricApiTest.CounterAggregationTest`, `UpDownCounterAggregationTest`, `ObservableCounterWithTagsAggregationTest`, `ObservableUpDownCounterWithTagsAggregationTest`?

Reply:
Instead of adding new tests, we have an existing test that could cover things. Basically, just add an additional `counter.Add(...)` here:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/a5e5ed8705e46d63102fb3026df6c891a3ea61fd/test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs#L46-L57

That said, the tests you have added do have the benefit of being closer to the unit of code _actually_ being tested - i.e., the `AggregatorStore`. I'd be content with simply removing the word `Counter` from your tests. Maybe just name it something like `LookupAggregatorStoreTest`. `LookupAggregatorStore` is the name of the private method that's actually doing the work being tested.

In the set up of your test you do:
```csharp
var aggregatorStore = new AggregatorStore("test", AggregationType.LongSumIncomingDelta, AggregationTemporality.Cumulative, 1024, Metric.DefaultHistogramBounds);
```

We _could_ use `InlineData` to introduce test cases with different permutations of `AggregationType`, `AggregationTemporality`, etc, but I do not feel strongly about this since varying these things do not affect the aggregator store lookup.